internal/internal.go

```diff
@@ -505,3 +509,10 @@ func getParsedPath(req *ext_authz.CheckRequest) []interface{} {
 	}
 	return parsedPathInterface
 }
+
+func getParsedBody(req *ext_authz.CheckRequest) map[string]interface{} {
+	body := req.GetAttributes().GetRequest().GetHttp().GetBody()
+	var data map[string]interface{}
+	json.Unmarshal([]byte(body), &data)
```

Comment:
Need to call `UseNumber` on the JSON decoder here otherwise there is potential for loss of precision on large integers. If you want, you can use the util package in OPA which does it for you.

Reply:
Used `util.Unmarshal`